regions/io/stcs/connect.py

```diff
@@ -0,0 +1,57 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from astropy.utils.data import get_readable_fileobj
+
+from regions.core import Region, Regions
+from regions.core.registry import RegionsRegistry
+
+__all__ = []
+
+
+@RegionsRegistry.register(Region, 'identify', 'stcs')
+@RegionsRegistry.register(Regions, 'identify', 'stcs')
+def is_stcs(methodname, filepath):
+    """
+    Identify an STC-S region file.
+
+    Parameters
+    ----------
+    methodname : {'read', 'write'}
+        The method name called that needs auto-identification.
+
+    filepath : str
+        The path to the file.
+
+    Returns
+    -------
+    result : bool
+        Returns `True` if the given file is an STC-S region file.
+    """
+    all_exten = ('.stcs', '.stc', '.stcs.txt', '.stc.txt')
```

Comment:
Do any of these files exist in the wild?  I'm happy enough to establish this as a new filename convention otherwise.